Author: Nilesh Patra <npatra974@gmail.com>
Description: Fix sphinx conf.py
Last-Update: Sun Sep 26 20:56:19 2021 +0530

--- a/doc/conf.py
+++ b/doc/conf.py
@@ -1,27 +1,27 @@
 # -*- coding: utf-8 -*-
 
 # General information about the project.
-project = u'patsy'
-copyright = u'2011-2015, Nathaniel J. Smith'
+project = 'patsy'
+copyright = '2011-2015, Nathaniel J. Smith'
 
 import sys
-print "python exec:", sys.executable
-print "sys.path:", sys.path
+print("python exec:", sys.executable)
+print("sys.path:", sys.path)
 try:
     import numpy
-    print "numpy: %s, %s" % (numpy.__version__, numpy.__file__)
+    print("numpy: %s, %s" % (numpy.__version__, numpy.__file__))
 except ImportError:
-    print "no numpy"
+    print("no numpy")
 try:
     import matplotlib
-    print "matplotlib: %s, %s" % (matplotlib.__version__, matplotlib.__file__)
+    print("matplotlib: %s, %s" % (matplotlib.__version__, matplotlib.__file__))
 except ImportError:
-    print "no matplotlib"
+    print("no matplotlib")
 try:
     import IPython
-    print "ipython: %s, %s" % (IPython.__version__, IPython.__file__)
+    print("ipython: %s, %s" % (IPython.__version__, IPython.__file__))
 except ImportError:
-    print "no ipython"
+    print("no ipython")
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
@@ -67,9 +67,9 @@
 # Undocumented trick: if we def setup here in conf.py, it gets called just
 # like an extension's setup function.
 def setup(app):
-    app.add_javascript("show-code.js")
-    app.add_javascript("facebox.js")
-    app.add_stylesheet("facebox.css")
+    app.add_js_file("show-code.js")
+    app.add_js_file("facebox.js")
+    app.add_css_file("facebox.css")
 
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ['_templates']
@@ -206,8 +206,8 @@
 # Grouping the document tree into LaTeX files. List of tuples
 # (source start file, target name, title, author, documentclass [howto/manual]).
 latex_documents = [
-  ('index', 'patsy.tex', u'patsy Documentation',
-   u'Nathaniel J. Smith', 'manual'),
+  ('index', 'patsy.tex', 'patsy Documentation',
+   'Nathaniel J. Smith', 'manual'),
 ]
 
 # The name of an image file (relative to this directory) to place at the top of
